assets/.jshintrc

```diff
@@ -3,25 +3,33 @@
   "browser": true,
   "esnext": true,
   "bitwise": true,
-  "camelcase": true,
+  "camelcase": false,
   "curly": true,
   "eqeqeq": true,
   "immed": true,
   "indent": 2,
   "latedef": true,
-  "newcap": true,
+  "newcap": false,
   "noarg": true,
+  "quotmark": false,
   "regexp": true,
-  "undef": true,
-  "unused": true,
+  "smarttabs": true,
   "strict": true,
+  "sub" : true,
   "trailing": true,
```

Comment:
Do you mean the `"trailing" : true`?  Agree, I think that jscs should be responsible for that.  Added it in the config in my [jscs branch](https://github.com/openshift/origin/compare/master...benjaminapetersen:bpeterse/bug/3519-jscs).